cli/src/cmd/mod.rs

```diff
@@ -94,6 +92,7 @@ If you are in a subdirectory in a Git repository, try adding `--root .`"#,
     } else if output.is_unchanged() {
         println!("no files changed, compilation skipped.");
     } else {
+        println!("{}", output);
```

Comment:
This addresses the warnings issue right? Does it still respect ignored error codes?

Reply:
yes, at this point we know there aren't any errors so this will only print warnings without those that are ignored.